src/Microsoft.DotNet.XHarness.Tests.Runners.Xunit/Microsoft.DotNet.XHarness.Tests.Runners.Xunit.csproj

```diff
@@ -1,38 +1,30 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <IsPackable>true</IsPackable>
-    <TargetFrameworks>netstandard2.1;netcoreapp3.1</TargetFrameworks>
-    <!-- Mono.Options is apparently not strong-name signed -->
-    <NoWarn>CS8002;</NoWarn>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Microsoft.DotNet.XUnitExtensions" Version="5.0.0-beta.20254.1" />
-    <PackageReference Include="Mono.Options" Version="6.6.0.161" />
-    <PackageReference Include="xunit.analyzers" Version="0.10.0" />
-    <PackageReference Include="xunit.extensibility.execution" Version="2.4.1" />
-    <PackageReference Include="xunit.runner.utility" Version="2.4.1" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <EmbeddedResource Include="xUnit\NUnit3Xml.xslt">
-      <XlfSourceFormat></XlfSourceFormat>
-      <XlfOutputItem></XlfOutputItem>
-    </EmbeddedResource>
-    <EmbeddedResource Include="xUnit\NUnitXml.xslt">
-      <XlfSourceFormat></XlfSourceFormat>
-      <XlfOutputItem></XlfOutputItem>
-    </EmbeddedResource>
-  </ItemGroup>
-
-  <ItemGroup>
-    <Compile Include="..\Microsoft.DotNet.XHarness.Common\XmlResultJargon.cs">
-      <Link>XmlResultJargon.cs</Link>
-    </Compile>
-    <Compile Include="..\Microsoft.DotNet.XHarness.iOS.Shared\Execution\Mlaunch\EnviromentVariables.cs">
-      <Link>EnviromentVariables.cs</Link>
-    </Compile>
-  </ItemGroup>
-
-</Project>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <IsPackable>true</IsPackable>
+    <TargetFrameworks>netstandard2.1;netcoreapp3.1</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.XUnitExtensions" Version="5.0.0-beta.20254.1" />
+    <PackageReference Include="xunit.analyzers" Version="0.10.0" />
+    <PackageReference Include="xunit.extensibility.execution" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.utility" Version="2.4.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="NUnit3Xml.xslt">
```

Comment:
Do we need the NUnit XSLT?

Reply:
Yes, used by xamarin-macios to get from xunit to nunit results.